bash/.bash_profile

```diff
@@ -17,53 +21,65 @@ if [ -d '/Applications/Visual Studio Code.app' ]; then
   export PATH="/Applications/Visual Studio Code.app/Contents/Resources/app/bin:$PATH"
 fi
 
-# Hide the annoying Bash/Zsh deprecation warning
-export BASH_SILENCE_DEPRECATION_WARNING=1
+# This stuff is only needed in interactive shells
+if [[ $- == *i* ]]; then
```

Comment:
Good consolidation! Moving all interactive shell configs under this check improves performance for non-interactive shells.